relayer/relayer_test.go

```diff
@@ -72,6 +88,13 @@ func TestCheckSupportedDestination(t *testing.T) {
 				supportedDestinations: set.Set[ids.ID]{
 					id1: {},
 				},
+				globalConfig: config.Config{
+					DestinationSubnets: []*config.DestinationSubnet{
+						{
+							BlockchainID: id1.String(),
+						},
+					},
+				},
```

Comment:
Would it be too much trouble to structure the `globalConfig` as we do in the real config, with `SupportedDestinations` inside a `SourceSubnet` which we then validate/parse?

Reply:
This has been reverted